tests/operator_test.go

```diff
@@ -1138,6 +1140,40 @@ spec:
 			deleteAllKvAndWait(true)
 		})
 
+		It("should create non-namespaces resources without owner references", func() {
+			crd, err := virtClient.ExtensionsClient().ApiextensionsV1beta1().CustomResourceDefinitions().Get("virtualmachineinstances.kubevirt.io", metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(crd.ObjectMeta.OwnerReferences).To(HaveLen(0))
+		})
+
+		It("should remove owner references on non-namespaces resources when updating a resource", func() {
+			By("adding an owner reference")
+			origCRD, err := virtClient.ExtensionsClient().ApiextensionsV1beta1().CustomResourceDefinitions().Get("virtualmachineinstances.kubevirt.io", metav1.GetOptions{})
+			crd := origCRD.DeepCopy()
+			crd.OwnerReferences = []metav1.OwnerReference{*metav1.NewControllerRef(&v1.KubeVirt{ObjectMeta: metav1.ObjectMeta{Name: "kubevirt", UID: "a185f8c3-3f38-4b89-a8cc-80f3731f7ff9"}}, v1.KubeVirtGroupVersionKind)}
```

Comment:
Is the UID a real ID or some randomly generated one?

Reply:
A random UID.